src/tools/dev/scripts/bv_support/bv_anari.sh

```diff
@@ -53,7 +53,7 @@ function bv_anari_info
     export ANARI_SHORT_VERSION=${ANARI_SHORT_VERSION:-"0.13"}
     export ANARI_FILE=${ANARI_FILE:-"ANARI-SDK-${ANARI_VERSION}.tar.gz"}
     export ANARI_COMPATIBILITY_VERSION=${ANARI_SHORT_VERSION}
-    export ANARI_URL=${ANARI_URL:-"https://github.com/KhronosGroup/ANARI-SDK/archive/refs/tags/v0.13.1.tar.gz"}
+    export ANARI_URL=""
```

Comment:
Is setting `ANARI_URL` to empty string intentional here?

Reply:
Ok, I see in this commit it is indeed intentional. Why not just remove setting of `ANARI_URL`? Or, does that cause problems?